qemu/tests/cfg/balloon_check.cfg

```diff
@@ -96,3 +96,9 @@
             services_up_timeout = 30
             sub_test_after_balloon_enlarge_to_0.75 = "guest_suspend"
             # s3 s4 related params will configure in guest-os.cfg
+        - balloon_repeat:
+            only balloon_base
+            repeat_times = 5000
```

Comment:
This value is  inconsistent with commit message and description, please keep consistent.